### DISCUSSION TITLE
Lesson 5: Issue running first script with Brownie

### DISCUSSION THREAD

**Discussion by Daruma1800** [smartcontractkit/full-blockchain-solidity-course-py#742] [Q&A] [answered]
Hi @cromewar @PatrickAlphaC , I've been stuck on the following command "brownie run scripts/deploy.py" and getting the below error (Timestamp: (4:34:26).  Any idea what is causing this issue? I am also using WSL and have installed the virutalenv. I made sure to include my node/ganache/brownie versions to help in any way!
![image](https://user-images.githubusercontent.com/94502087/148692399-6f16f1df-ddc9-4359-8f6b-56c7c0ab0c43.png)

**Comment by cromewar** [comment, marked as answer]
Hello there, as dar as I know that node version has problems with web3 could you please change to a lts version of node? (16 should be the one )

**Reply by Daruma1800** [reply]
I installed node version 16.3.0 and still getting the same error: 
![image](https://user-images.githubusercontent.com/94502087/148697820-4f7858d1-3b86-4517-a45d-c095fa5c85be.png)

**Reply by Daruma1800** [reply]
@cromewar okay I just restarted and reinstalled everything and got through!

**Reply by cromewar** [reply]
Jum that's strange, I glad it's solved now tough.

**Comment by Daruma1800** [comment]
Run: Nvm install 16.3.0

Node - - version and you should be good to go. A

Then re-run your command

**Reply by cromewar** [reply]
Nice @Daruma1800
